server/impl/src/main/java/com/walmartlabs/concord/server/security/ldap/LdapUserInfoProvider.java

```diff
@@ -78,13 +78,8 @@ public UUID create(String username, String userDomain, String displayName, Strin
             // this will result in a 401 response with an empty body anyway
             throw new ConcordApplicationException("Automatic creation of users is disabled.");
         }
-
-        UserInfo info = getInfo(null, username, userDomain);
-        if (info == null) {
-            throw new ConcordApplicationException("User '" + username + "' with domain '" + userDomain + "' not found in LDAP");
-        }
-
-        return userDao.insertOrUpdate(info.username(), info.userDomain(), info.displayName(), info.email(), UserType.LDAP, roles);
+        
+        return userDao.insertOrUpdate(username, userDomain, displayName, email, UserType.LDAP, roles);
```

Comment:
With this change the `UserInfoProvider#create` method becomes redundant - both implementations are the same (just a DAO call). I think we should remove the method and call `userDao.insertOrUpdate(username, userDomain, displayName, email, UserType.LDAP, roles)` in `UserManager`.

Reply:
done